roles/power_aix_bootstrap/vars/main.yml

```diff
@@ -1,5 +1,7 @@
-# Copyright (c) IBM Corporation 2020
-pkgtype: "yum"
+# Copyright (c) IBM Corporation 2021
+pkgtype: ""
```

Comment:
do you really need `pkgtype` in here. this is a required parameter for the bootstrap role. before the default value
was `yum`, so if you did not enter anythin for the `pkgtype` var for the role it would default to yum.
here you made it empty, which i assume you did not want to make `yum` bootstrap the default behavior.

probably just indicate in the updated README.md for the bootstrap role that `pkgtype` is required